client/templates/includes/editCustomStoichSpecie.pug

```diff
@@ -1,10 +1,10 @@
 div
   div.inline
-    button.btn.btn-outline-secondary(class='btn-sm', data-hook='decrement') -
-  span.custom(data-hook="ratio")
+    button.btn.btn-outline-secondary(class='btn-sm', data-hook='decrement') -id="", 
+  span.custom(id="ratio", data-hook="ratio")
   div.inline
-    button.btn.btn-outline-secondary(class='btn-sm', data-hook='increment') +
-  select(data-hook="select-stoich-specie")
-    span.message.message-below.message-error(data-hook="message-container")
-      p(data-hook="message-text")
-  button.btn.btn-outline-secondary.custom(class='btn-sm', data-hook='remove') X
\ No newline at end of file
+    button.btn.btn-outline-secondary(class='btn-sm', data-hook='increment') +id="", 
```

Comment:
This id attribute should be inside the parens, the '+' is the text for the button